src/test/java/teammates/test/cases/webapi/GetInstructorActionTest.java

```diff
@@ -23,13 +29,53 @@ protected String getRequestMethod() {
     @Test
     @Override
     protected void testExecute() throws Exception {
-        // TODO
+        InstructorAttributes instructor1OfCourse1 = typicalBundle.instructors.get("instructor1OfCourse1");
+        FeedbackSessionAttributes feedbackSessionAttributes = typicalBundle.feedbackSessions.get("session1InCourse1");
+
+        loginAsInstructor(instructor1OfCourse1.googleId);
+
+        ______TS("Not enough parameters");
+        verifyHttpParameterFailure();
+        verifyHttpParameterFailure();
+        verifyHttpParameterFailure(Const.ParamsNames.COURSE_ID, feedbackSessionAttributes.getCourseId());
+        verifyHttpParameterFailure(Const.ParamsNames.FEEDBACK_SESSION_NAME,
+                feedbackSessionAttributes.getFeedbackSessionName());
+        verifyHttpParameterFailure(Const.ParamsNames.COURSE_ID, feedbackSessionAttributes.getCourseId(),
+                Const.ParamsNames.FEEDBACK_SESSION_NAME, feedbackSessionAttributes.getFeedbackSessionName());
+
+        ______TS("typical success case");
+        String[] params = {
+                Const.ParamsNames.COURSE_ID, feedbackSessionAttributes.getCourseId(),
+                Const.ParamsNames.FEEDBACK_SESSION_NAME, feedbackSessionAttributes.getFeedbackSessionName(),
+                Const.ParamsNames.INTENT, Intent.INSTRUCTOR_SUBMISSION.toString(),
+        };
+
+        GetInstructorAction a = getAction(params);
+        JsonResult r = getJsonResult(a);
```

Comment:
Since `JsonResult r` isn't really used directly here, you can just chain the method calls together to yield a `InstructorData`, e.g. `InstructorData response = (InstructorData) getJsonResult(action).getOutput()`

Reply:
It is also used to check for the status code! (have modified the rest)